src/main/java/Parser.java

```diff
@@ -94,7 +95,7 @@ public void parseCommand(String input, TaskList taskList) throws DukeException {
 
             taskList.addTask(newEvent);
 
-            ui.showTaskAdded(newEvent, taskList);
+            return ui.showTaskAdded(newEvent, taskList);
         }
```

Comment:
Good use of OOP and clean code! But reading through this it might be slightly unclear, maybe you can refactor this part to use case and switch statements?